src/services/PrivateRoutes.js

```diff
@@ -0,0 +1,13 @@
+import React  from 'react';
+import {Outlet,Navigate} from 'react-router-dom';
+
+
+const PrivateRoutes = () => {
+    let auth  = {'token':false}
+
+    return (
+       auth.token ? <Outlet/> : <Navigate to = '/signup'/>
+    );
+};
```

Comment:
Isn't this always false? You should look at [`supabase.auth` ](https://supabase.com/docs/reference/javascript/auth-user)

Reply:
yes, it is always false, I wanted to test. Instead of the token, I will use the user data.
users ? <Outlet/> : <Navigate to = '/signup'/>